terraform/state-persistence.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "gt-site-cluster" {
```

Comment:
This resource is setup to interact with Terraform state pushed and pulled primarily by another project. We should omit this resource and manage the Terraform state for this project via `terraform remote config` and `terraform remote push`:

```bash
            terraform remote config \
                      -backend="s3" \
                      -backend-config="region=us-east-1" \
                      -backend-config="bucket=${RF_SETTINGS_BUCKET}" \
                      -backend-config="key=terraform/state" \
                      -backend-config="encrypt=true"

...

                    terraform get -update
                    terraform plan \
                              -var-file="${RF_SETTINGS_BUCKET}.tfvars" \
                              -out="${RF_SETTINGS_BUCKET}.tfplan"

...

                    terraform apply "${RF_SETTINGS_BUCKET}.tfplan"
                    terraform remote push
```